manifests/windows/install.pp

```diff
@@ -12,8 +12,17 @@
     default   => $::puppet_agent::arch
   }
 
+  if $::puppet_agent::is_pe {
```

Comment:
Also not explicitly supported, we'll have to verify when we enable running on Puppet 4 agents.